src/Http/Http.Extensions/src/RequestDelegateFactory.cs

```diff
@@ -1565,16 +1566,24 @@ private static Expression BindParameterFromValue(ParameterInfo parameter, Expres
             //      wasParamCheckFailure = true;
             //      Log.RequiredParameterNotProvided(httpContext, "TypeOfValue", "param1");
             // }
+
+            Expression checkRequiredExpression = Expression.Empty();
+
+            if (!parameter.ParameterType.IsValueType)
```

Comment:
For `StringValues` parameter we should skip null-checking, because of `value[some_unpresent_parameter]` returns `StringValue.Empty`, and `StringValue.Empty` is equal to `null` ([StringValues.cs source](https://github.com/dotnet/runtime/blob/621cd59436cb29cab4b1162409ae0947c4bd780d/src/libraries/Microsoft.Extensions.Primitives/src/StringValues.cs#L709)):

https://github.com/dotnet/runtime/blob/621cd59436cb29cab4b1162409ae0947c4bd780d/src/libraries/Microsoft.Extensions.Primitives/src/StringValues.cs#L705-L710